### DISCUSSION TITLE
Add raw compose deployment option for Docker compose applications

### DISCUSSION THREAD

**Discussion by Fheuef** [coollabsio/coolify#3208] [Feature Requests]
While trying to deploy an application using docker compose with custom network options (that were conflicting with some coolify magic), I discovered that the "Raw compose deployment" option mentionned in the documentation was nowhere to be found in the compose app menus. I ended up having to create a new git project with a single compose file to access the raw option.

It would be nice (and less confusing) to have that option available directly in the settings of a Compose application.

**Comment by peaklabs-dev** [comment]
Some fixes will be added in the next version

**Reply by nicanordlc** [reply]
Yeah, it would be nice if we could have more of these options for a `Docker Compose Empty` type of service.
